Library/Homebrew/extend/os/linux/hardware/cpu.rb

```diff
@@ -1,6 +1,16 @@
 module Hardware
   class CPU
     class << self
+      OPTIMIZATION_FLAGS_LINUX = {
```

Comment:
`LINUX_OPTIMIZATION_FLAGS` would be more consistent with `PPC_OPTIMIZATION_FLAGS`. I wonder if it's worth just overriding the `dunno` in OPTIMIZATION_FLAGS instead? The other values look good?